packages/build/src/npm-packages/publish.ts

```diff
@@ -71,40 +85,35 @@ export function publishNpmPackages(
     );
 
     spawnSync('git', ['push', '--follow-tags'], commandOptions);
-  }
-}
 
-export function markBumpedFilesAsAssumeUnchanged(
-  packages: LernaPackageDescription[],
-  assumeUnchanged: boolean,
-  spawnSync: typeof spawnSyncFn = spawnSyncFn
-): void {
-  const filesToAssume = [
-    path.resolve(PROJECT_ROOT, 'lerna.json'),
-    path.resolve(PROJECT_ROOT, 'package.json'),
-    path.resolve(PROJECT_ROOT, 'package-lock.json'),
-  ];
-  for (const { location } of packages) {
-    filesToAssume.push(path.resolve(location, 'package.json'));
-  }
-
-  for (const f of filesToAssume) {
+    // Merge the release branch into main
+    spawnSync('git', ['checkout', 'origin/main'], commandOptions);
     spawnSync(
       'git',
-      [
-        'update-index',
-        assumeUnchanged ? '--assume-unchanged' : '--no-assume-unchanged',
-        f,
-      ],
-      {
-        stdio: 'inherit',
-        cwd: PROJECT_ROOT,
-        encoding: 'utf8',
-      },
-      true
+      ['merge', '--squash', `origin/release/${mongoshVersion}`],
+      commandOptions
+    );
+    spawnSync(
```

Comment:
I guess this essentially merges the PR but wondering if it'd be worth using GitHub CLI here instead to i.e. have reference to the original PR naming when merging. Don't know how easy it is with how our GitHub Actions Bot token works

Reply:
Yeah, right now this pushes from evergreen, right? Do we even have permissions for that?

Reply:
Depends on the token - the devtools bot token does have permissions for that, but I'd really like to avoid pushing to main if there's an open PR we could merge.

Reply:
Hm. We can either make the merging of the PR manual, [based on a cron job like in Compass](https://github.com/mongodb-js/compass/blob/4916ea697e71f092fef826392a1a1024d72f1feb/.github/workflows/merge-bump-packages-pr.yaml#L29), or try to set it up in evergreen, i.e. install the GitHub CLI inside evergreen and then use GITHUB_TOKEN (not sure what permissions this is) to merge the created PR